src/Network/Receive/ServerType0.pm

```diff
@@ -2861,143 +2270,6 @@ sub character_equip {
-			}
-		} else {
-			if ($config{storageAuto_password} eq '') {
-				my $input = $interface->query(T("Please enter your storage password."), isPassword => 1);
-				if (!defined($input)) {
-					return;
-				}
-				configModify('storageAuto_password', $input, 1);
-				message TF("Storage password set to: %s\n", $input), "success";
-			}
-		}
-
-		my @key = split /[, ]+/, $config{storageEncryptKey};
-		if (!@key) {
-			error (($args->{switch} eq '023E') ?
-				T("Unable to send character password. You must set the 'storageEncryptKey' option in config.txt or servers.txt.\n") :
-				T("Unable to send storage password. You must set the 'storageEncryptKey' option in config.txt or servers.txt.\n"));
-			return;
-		}
-		my $crypton = new Utils::Crypton(pack("V*", @key), 32);
-		my $num = ($args->{switch} eq '023E') ? $config{charSelect_password} : $config{storageAuto_password};
-		$num = sprintf("%d%08d", length($num), $num);
-		my $ciphertextBlock = $crypton->encrypt(pack("V*", $num, 0, 0, 0));
-		$messageSender->sendStoragePassword($ciphertextBlock, 3);
-
-	} elsif ($args->{flag} == 8) {	# apparently this flag means that you have entered the wrong password
-									# too many times, and now the server is blocking you from using storage
-		error T("You have entered the wrong password 5 times. Please try again later.\n");
-		# temporarily disable storageAuto
-		$config{storageAuto} = 0;
-		my $index = AI::findAction('storageAuto');
-		if (defined $index) {
-			AI::args($index)->{done} = 1;
-			while (AI::action ne 'storageAuto') {
-				AI::dequeue;
-			}
-		}
-	} else {
-		debug(($args->{switch} eq '023E') ?
-			"Character password: unknown flag $args->{flag}\n" :
-			"Storage password: unknown flag $args->{flag}\n");
-	}
-}
-
-# TODO
-sub storage_password_result {
-	my ($self, $args) = @_;
-
-	# TODO:
-    # STORE_PASSWORD_EMPTY =  0x0
-    # STORE_PASSWORD_EXIST =  0x1
-    # STORE_PASSWORD_CHANGE =  0x2
-    # STORE_PASSWORD_CHECK =  0x3
-    # STORE_PASSWORD_PANALTY =  0x8
-
-	if ($args->{type} == 4) { # STORE_PASSWORD_CHANGE_OK =  0x4
-		message T("Successfully changed storage password.\n"), "success";
-	} elsif ($args->{type} == 5) { # STORE_PASSWORD_CHANGE_NG =  0x5
-		error T("Error: Incorrect storage password.\n");
-	} elsif ($args->{type} == 6) { # STORE_PASSWORD_CHECK_OK =  0x6
-		message T("Successfully entered storage password.\n"), "success";
-	} elsif ($args->{type} == 7) { # STORE_PASSWORD_CHECK_NG =  0x7
-		error T("Error: Incorrect storage password.\n");
-		# disable storageAuto or the Kafra storage will be blocked
-		configModify("storageAuto", 0);
-		my $index = AI::findAction('storageAuto');
-		if (defined $index) {
-			AI::args($index)->{done} = 1;
-			while (AI::action ne 'storageAuto') {
-				AI::dequeue;
-			}
-		}
-	} else {
-		#message "Storage password: unknown type $args->{type}\n";
-	}
-
-	# $args->{val}
-	# unknown, what is this for?
-}
-
-sub initialize_message_id_encryption {
```

Comment:
initialize_message_id_encryption was removed from ServerType0 and added to Receive, but it's still in Sakexe_0